streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/runner/EnvInitializer.java

```diff
@@ -70,8 +70,21 @@ public class EnvInitializer implements ApplicationRunner {
           "^streampark-flink-shims_flink-(1.12|1.13|1.14|1.15|1.16)_(2.11|2.12)-(.*).jar$",
           Pattern.CASE_INSENSITIVE | Pattern.DOTALL);
 
+  // We introduced it for https://github.com/apache/incubator-streampark/issues/2014
+  private boolean shouldSkip() {
+    // Use try ... catch to avoid NPE or AIOB.
+    try {
+      return "test".equals(context.getEnvironment().getActiveProfiles()[0]);
+    } catch (Exception e) {
+      return false;
+    }
+  }
+
   @Override
   public void run(ApplicationArguments args) throws Exception {
+    if (shouldSkip()) {
```

Comment:
how about?
```
Optional<String> profile = Arrays.stream(context.getEnvironment().getActiveProfiles()).findFirst();
    if ("test".equals(profile.orElse(null))) {
        return;
    }
```

Reply:
Thx for the commentt @wolfboys .
I'd like it.
Updated  & made optimization change in the EnvInitializer.
PTAL in your free time~.

Reply:
> Thx for the commentt @wolfboys . I'd like it. Updated & made optimization change in the EnvInitializer. PTAL in your free time~.

👌 good job.